docs/users-guide/03-basic-exploration.md

```diff
@@ -69,6 +69,25 @@ Searches take into account items’ titles, descriptions, and other metadata —
 
 ![Search results](./images/basic-exploration/search-results.png)
 
+## Bookmarks
+
+**Bookmarks** are a way to quickly get back to things you visit frequently (or have been working on recently). Bookmarked items show up in the main navigation sidebar above [collections][collections].
+
+To bookmark an item, look for the ribbon icon. If you don't immediately see the ribbon, open up the settings for that item. You can bookmark:
```

Comment:
```suggestion
To bookmark an item, look for the **ribbon** icon. If you don't immediately see the ribbon, open up the settings for that item. You can bookmark:
```

Reply:
I don't think we're always consistent about bolding the description of an icon (eg "pencil icon", etc), but it seems like we usually do?